weave-js/tsconfig.json

```diff
@@ -16,7 +16,8 @@
       "scripthost",
       "esnext.asynciterable",
       "es2017.object",
-      "ES2019"
+      "ES2019",
+      "es2023"
```

Comment:
what does this ignore?

Reply:
It doesn't ignore anything but rather allows for usage of JS standard in the ecmascript 2023 standard. I'm specifically using `findLastIndex`